nginx.tmpl

```diff
@@ -12,14 +12,24 @@ events {
 http {
   proxy_cache_path /tmp/nginx_cache levels=1:2 keys_zone=image_cache:10M max_size=100M inactive=40d;
 
+  # This prevents internal IP addresses from leaking when providing an empty host header
+  # requests where the Host header does not match *.ndla.no will be closed (nginx closes the connection when returning a 444)
+  server {
+    listen 79;
```

Comment:
Det ser ut til at dette problemet bare oppstår ved en redirect, så tror ikke det er et problem når man allerede er på https. Skader uansett ikke å legge på en tilsvarende blokk for port 80